core/src/main/java/org/infinispan/marshall/core/ExternallyMarshallable.java

```diff
@@ -2,18 +2,18 @@
 
 import java.io.Serializable;
 
-// Temporary class, should become active by default when testing
 final class ExternallyMarshallable {
 
    private ExternallyMarshallable() {
       // Static class
    }
 
-   public static boolean isAllowed(Object obj) {
+   static boolean isAllowed(Object obj) {
       String pkg = obj.getClass().getPackage().getName();
       boolean isBlackList =
             obj instanceof Serializable
             && isMarshallablePackage(pkg)
+            && !(obj instanceof ExternalPojo)
             && !isWhiteList(obj.getClass().getName());
```

Comment:
@galderz I was thinking about the `ExternalPojo` javadoc, which says `One such example are pojos used in testing, but not limited to,` and I wasn't sure what the actual limits are ;)